cl/corpus_importer/management/commands/update_opinions_order.py

```diff
@@ -0,0 +1,96 @@
+from django.db.models import Count, Q
+
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.search.models import SOURCES, Opinion, OpinionCluster
+
+
+def sort_harvard_opinions(options) -> None:
+    """Sort harvard opinions
+
+    We assume that harvard data is already ordered, we just need to fill
+    the order field in each opinion
+
+    The harvard importer created the opinions in order of appearance in the file
+
+    :param options: dict of arguments skip until and limit if given
+    :return: None
+    """
+
+    skip_until = options.get("skip_until", None)
+    limit = options.get("limit", None)
+
+    base_filter = (
+        OpinionCluster.objects.exclude(filepath_json_harvard="")
+        .annotate(opinions_count=Count("sub_opinions"))
+        .filter(opinions_count__gt=1)
+    )
+
+    if skip_until:
+        base_filter &= Q(pk__gte=skip_until)
+
+    harvard_clusters = (
+        OpinionCluster.objects.annotate(opinions_count=Count("sub_opinions"))
+        .filter(base_filter)
```

Comment:
I don't *think* this will do what you want it to. I think you'll be passing a QuerySet into a filter, which does a subquery. It might work, but I don't think it's efficient. It's worth checking by printing the query. 

Some other things:
 - It's also confusing that you do the annotation twice, here and above. 
 - If we're going to iterate over all the subopinions, we should do a select_related.

Putting this all together, I think you can delete everything above and replace it with something like:

```suggestion
    harvard_clusters = (
        OpinionCluster.objects.exclude(filepath_json_harvard="")
        .select_related("sub_opinions")
        .annotate(opinions_count=Count("sub_opinions"))
        .filter(opinions_count__gt=1)
        .order_by("id")
    )
    if skip_until:
        harvard_clusters = harvard_clusters.filter(pk__gte=skip_until)
```

Would that check all the boxes?